### DISCUSSION TITLE
Raw-Value incorrect

### DISCUSSION THREAD

**Discussion by Ingo-Knito** [jomjol/AI-on-the-edge-device#2703] [General]
### The Problem

The raw value is interpreted incorrectly at irregular intervals.
I have not yet recognised a rule. Even a reboot does not help.

Recognising the digits works correctly (see screenshot). But with the raw value, the penultimate decimal place is repeatedly decremented by one, which then leads to a negative value.

### Version

15.3.0

### Logfile

```shell
[0d14h31m32s] 2023-11-16T00:21:39	<ERR>	[POSTPROC] main: Raw: 205340.6, Value: , Status: Neg. Rate - Read:  - Raw: 205340.6 - Pre: 205348.8 
[0d17h16m32s] 2023-11-16T03:06:39	<ERR>	[POSTPROC] main: Raw: 205349.2, Value: , Status: Neg. Rate - Read:  - Raw: 205349.2 - Pre: 205358.4 
[0d21h46m32s] 2023-11-16T07:36:37	<ERR>	[POSTPROC] main: Raw: 205369.2, Value: , Status: Neg. Rate - Read:  - Raw: 205369.2 - Pre: 205377.8
```

### Screenshots

![Overview](https://github.com/jomjol/AI-on-the-edge-device/assets/39132657/54584bf1-bb83-4fec-8566-3ec654e2e208)

![Recognition](https://github.com/jomjol/AI-on-the-edge-device/assets/39132657/a9cee40b-0b8f-410e-8be8-b6d3d17645d3)

**Comment by caco3** [comment]
It is nearly impossible for the device to be always 100% correct.
Eg. the 3th digit from the right should be read as a 6.9. How ever it also could be decided that the digit already is at 7.
That's why there are those validations regarindg max rate and neg rate.

What you can do:
- Try a different model
- Adjust the max and neg rates
- Move the ROIs slightly
Please have a look on the documentation at https://jomjol.github.io/AI-on-the-edge-device-docs/

**Reply by Ingo-Knito** [reply]
Thanks for the tips.

In my opinion, the software does a great job - in fact 100%!
But since the recognition works perfectly here - in my opinion - I was surprised by the further interpretation.

Then I'll try the first two points - I know the documentation by heart :-)
